packages/editor/src/components/editor/canvas/Canvas.tsx

```diff
@@ -5,22 +5,23 @@ import { useAppContext } from '../../../context/useData';
 import { DropZone, type DropZoneProps } from './DropZone';
 import type { Component, ComponentData } from '@axonivy/form-editor-protocol';
 import { CANVAS_DROPZONE_ID } from '../../../data/data';
+import { forwardRef } from 'react';
 
 type CanvasProps = {
   config: Config;
 };
 
-export const Canvas = ({ config }: CanvasProps) => {
+export const Canvas = forwardRef<HTMLDivElement, CanvasProps>(({ config }, ref) => {
   const { ui, data } = useAppContext();
   return (
-    <div className='canvas' data-help-paddings={ui.helpPaddings} data-responsive-mode={ui.responsiveMode}>
+    <div className='canvas' data-help-paddings={ui.helpPaddings} data-responsive-mode={ui.responsiveMode} ref={ref}>
```

Comment:
can you just add an onclick method to the canvas itself? sure at least you would need a check if the click was not from a sub element. I think this would not make the code a little bit less complicated. 
E.g. with something like: https://stackoverflow.com/a/68067875

If this is not working, totally fine, go with your solution.

Reply:
You are right, if I simply do an onClick on the canvas div, it works as follows:
![grafik](https://github.com/axonivy/form-editor-client/assets/141223521/aa05ac38-47f4-497a-bb33-9eb267b177d9)

The only downside to this approach is that when I click on the grey area outside the canvas, the selection is not reset. It is only reset if I click in the white area between the components. What do you think, is that enough? I have the feeling that you are more likely to click in the grey area to reset the selection. But that's just a guess ✌️ 
![grafik](https://github.com/axonivy/form-editor-client/assets/141223521/17cef60f-c806-435f-8763-3040b0f86868)

Reply:
and if you set this onclick function to the resize area?

Reply:
nice, that did the trick 👍